hadoop/src/tesser/hadoop.clj

```diff
@@ -200,3 +200,68 @@
                   first)]
         (when (error? x) (throw (ex-info "Hadoop fold error" x)))
         x))))
+
+(defn identity-mapper
+  "Does nothing in the map step."
+  {::mr/source-as :vals
+   ::mr/sink-as   :vals}
+  [records]
+  (r/map identity records))
+
+(defn partition-randomly
+  ^long [key val ^long nparts]
```

Comment:
Can we get a docstring on this? Why are key and val ignored? Is ^long supposed to be a type hint on `partition-randomly`?